README.md

```diff
@@ -111,52 +58,15 @@ A local dev server will be created on
 
 Flask environment variables are configurable in `.flaskenv`
 
-### Run with Gunicorn
-
-In deployed environments the service is run with gunicorn. You can run the service locally with gunicorn to test
-
-First set the FLASK_ENV environment you wish to test eg:
-
-    export FLASK_ENV=dev
-
-Then run gunicorn using the following command:
-
-    gunicorn wsgi:app -c run/gunicorn/local.py
-
-### Build with Paketo
-
-[Pack](https://buildpacks.io/docs/tools/pack/cli/pack_build/)
-
-[Paketo buildpacks](https://paketo.io/)
-
-```pack build <name your image> --builder paketobuildpacks/builder:base```
-
-Example:
-
-```
-[~/work/repos/funding-service-design-assessment-store] pack build paketo-demofsd-app --builder paketobuildpacks/builder:base
-***
-Successfully built image paketo-demofsd-app
-```
-
-You can then use that image with docker to run a container
-
-```
-docker run -d -p 8080:8080 --env PORT=8080 --env FLASK_ENV=dev [envs] paketo-demofsd-app
-```
+## Paketo
+Paketo is used to build the docker image which gets deployed to our test and production environments. Details available [here](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-paketo.md)
 
 `envs` needs to include values for each of (set with `--env <varname>=<value>`):
 APPLICATION_STORE_API_HOST
 SENTRY_DSN
 GITHUB_SHA
 DATABASE_URL
 
-```
-docker ps -a
-CONTAINER ID   IMAGE                       COMMAND                  CREATED          STATUS                    PORTS                    NAMES
-42633142c619   paketo-demofsd-app          "/cnb/process/web"       8 seconds ago    Up 7 seconds              0.0.0.0:8080->8080/tcp   peaceful_knuth
-```
-
```

Comment:
The pipeline bit below is incorrect - we no longer use deploy to gov paas

Reply:
Have removed